Package.swift

```diff
@@ -46,7 +46,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/microsoft/plcrashreporter.git", from: "1.11.2"),
-        .package(url: "https://github.com/open-telemetry/opentelemetry-swift.git", exact: "1.6.0")
```

Comment:
once https://github.com/DataDog/opentelemetry-swift-packages/pull/17 is merged, we will take it back to 1.6.0.